userspace/libscap/scap.c

```diff
@@ -1680,3 +1680,8 @@ uint64_t scap_get_driver_schema_version(scap_t* handle)
 {
 	return handle->m_schema_version;
 }
+
+scap_swap_endian scap_get_swap_endian(scap_t* handle)
+{
+	return handle->m_reader->m_swap_endian;
+}
```

Comment:
How does this not explode in live mode? Or do we just never call it from live captures?